fighthealthinsurance/auth/auth_utils.py

```diff
@@ -0,0 +1,9 @@
+# Different clinics will have different "domains" and the django username will be the user visible username + a clinic name seperated by a panda unicode
+
+def validate_username(username: str) -> bool:
```

Comment:
What if we just used two model fields instead of one to store this? EG, `unique_together = ('username', 'domain')` instead of having only username be unique? Is there a security concern with why we don't want to do that or something?

Reply:
Yeah I think that's reasonable too. This just avoids us having to replace the django user model.

Reply:
What do you think about replacing the user model?

Reply:
Looking at the django docs it seems like migrating after launch is bad :(

Reply:
Ok poked at it some more -- changing post launch is doable (especially since we have no foriengkey relationships on users just yet) _but_ we'll still probably want to keep our "hacks" since the requirement that the username is unique